backend/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "typescript": "^5.4.2"
+  },
+  "dependencies": {
+    "fastify": "^4.26.2",
+    "json-schema-to-typescript": "^13.1.2"
```

Comment:
Pretty sure that json-schema-to-typescript can be a dev dependency instead of a regular one. But see comments about json-schema-to-typescript in general before fixing.